examples/14_dlesym_example.py

```diff
@@ -0,0 +1,274 @@
+variables which are not provided by standard data sources but can be computed
+from the standard variables.
+
+In this example you will learn:
+
+- How to instantiate the DLESyM model
+- How to use the model API to generate a forecast
+- How to use the output selection and regridding methods to select appropriate data
+- How to use the `DLESyMLatLon` model with earth2studio workflows
+"""
+
+# %%
+# Set Up
+# ------
+# The first step is fetching appropriate input data for the model. The ERA5 data sources
+# in earth2studio provide data on the lat/lon grid, so have two options:
+#  -  use the :py:class:`earth2studio.models.DLESyMLatLon` model. This version of DLESyM
+#     accepts inputs on the lat/lon grid and regrids them to the HEALPix grid internally,
+#     before returning the output regridded back to the lat/lon grid. This is the
+#     recommended approach for most users as it can be used directly with earth2studio
+#     data sources and workflows, since it performs regridding and pre-processing
+#     internally.
+#  -  use the :py:class:`earth2studio.models.DLESyM` model, and handle the regridding of
+#     input lat/lon data ourselves. Since the model uses some derived variables which
+#     are not provided by the data source, we would also need to prepare these derived
+#     variables ourselves.
+#
+# Let's load both of these models and inspect the expected input coordinates for each.
+# Also note the input and output variable set for each model.
+# %%
+import os
+
+os.makedirs("outputs", exist_ok=True)
+from dotenv import load_dotenv
+
+load_dotenv()  # TODO: make common example prep function
+import numpy as np
+import torch
+
+from earth2studio.data import ARCO
+from earth2studio.data.utils import fetch_data
+from earth2studio.models.px.dlesym import DLESyM, DLESyMLatLon
+
+device = "cpu" if not torch.cuda.is_available() else "cuda"
+
+# Create the data source
+data = ARCO()
+
+# Load the default model package which downloads the check point from NGC
+# We will instantiate both versions of the model to demonstrate usage of each.
+package = DLESyMLatLon.load_default_package()
+model_ll = DLESyMLatLon.load_model(package).to(device)
+package = DLESyM.load_default_package()
+model_hpx = DLESyM.load_model(package).to(device)
+
+in_coords_ll = model_ll.input_coords()
+in_coords_hpx = model_hpx.input_coords()
+print(
+    "DLESyM LatLon input coord shapes: ",
+    [(k, v.shape) for k, v in in_coords_ll.items()],
+)
+print(
+    "DLESyM HPX input coord shapes: ", [(k, v.shape) for k, v in in_coords_hpx.items()]
+)
+print("Lat-lon input variable names: ", in_coords_ll["variable"])
+print(
+    "Lat-lon output variable names: ", model_ll.output_coords(in_coords_ll)["variable"]
+)
+print("HEALPix input variable names: ", in_coords_hpx["variable"])
+print(
+    "HEALPix output variable names: ",
+    model_hpx.output_coords(in_coords_hpx)["variable"],
+)
+# %%
+# Making predictions, regridding, and selecting outputs
+# ---------------
```

Comment:
Sphinx Nit: "---------------" should be same length as title

Making predictions, regridding, and selecting outputs > Capitalize words like a title